src/r_data.c

```diff
@@ -1165,6 +1165,9 @@ boolean R_IsPatchLump (const int lump)
   const patch_t * patch;
   boolean result;
 
+  if (!strncasecmp("DOOMPRES", W_WadNameForLump(lump), 8))
```

Comment:
Can we get rid of the multiple WAD name checks and somehow check this only once? Like gamemode and gamemission?

Reply:
Also, how does this work together with the `-beta` command line switch?

Reply:
> Can we get rid of the multiple WAD name checks and somehow check this only once? Like gamemode and gamemission?

Yes, sure. This is unfinished.

> Also, how does this work together with the `-beta` command line switch?

Not currently.

Reply:
👍